lib/dor/update_marc_record_service.rb

```diff
@@ -58,7 +58,7 @@ def run_write_script(command)
         stderr_text = stderr.read
         
         if stdout_text.length > 0 || stderr_text.length > 0 then
-          raise "There was an error in writting marc_record file using the command #{command}\n#{stdout_text}\n#{stderr_text}"
+          raise "There was an error in writing marc_record file using the command #{command}\n#{stdout_text}\n#{stderr_text}"
```

Comment:
would probably either say "There was an error in writing the marc_record file..." or "error in writing marc_record file..." just so the sentence is consistently terse or not throughout, for easier reading.